### DISCUSSION TITLE
Is it possible run mastodon on other port then 443?

### DISCUSSION THREAD

**Discussion by EsherionM** [mastodon/mastodon#22776] [General]
I have multiple virtual machines at home and port 443 is already taken from there.
Is it possible to change the port from mastodon or could I remove and forget the mastodon server?

**Comment by EsherionM** [comment]
No answer means no. So i have deleted my virtual Server and forgot the mastodon project.

**Reply by Traction6131** [reply]
It totally is though, for example with docker and diffrent port mapping. This is a massively huge project, so you can't really expect fast responses.
